includes/classes/SupportMonitor/DBQueryMonitor.php

```diff
@@ -0,0 +1,173 @@
+<?php
+/**
+ * Database Queries Monitor. A submodule of Support Monitor to report heavy SQL queries executed on staging.
+ *
+ * @since  x.x
+ * @package 10up-experience
+ */
+
+namespace TenUpExperience\SupportMonitor;
+
+/**
+ * DBQueryMonitor class
```

Comment:
@felipeelia I'd love to see more comments describing what this class does and how it works in plain terms so there isn't any concern from installing this and assumptions of anything negative that may be happening with the queries. For example, I'd like to see a description of what the class checks for, that it only uses non-production environments, etc.

Reply:
@bengreeley I put part of that in the doc description (`A submodule of Support Monitor to report heavy SQL queries executed on staging.`). As we are still defining what is a "heavy SQL query", I'll revisit it later but you made me curious about the best location of that. Is it in the class doc block or in the file's one?

Reply:
I think this is good for now! I'm sure at some point we can improve documentation in general. Thank you